konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/build/extension/PythonPathsCommand.java

```diff
@@ -0,0 +1,243 @@
+                        type.name(), Arrays.toString(PythonPathsCommand.ListInstallationType.values()));
+        }
+    }
+
+    public static void registerInstallation() {
+
+    }
+
+    private static void listJavacppInstallations() {
+
+    }
+
+    private static void listPythonInstallations() {
+        System.out.println("\n----------------------------PYTHON INSTALLS----------------------------");
+        System.out.print(
+                findPythonInstallations().stream()
+                        .map(PythonPathsCommand::formatPythonInstallation)
+                        .collect(Collectors.joining(System.lineSeparator()))
+        );
+        System.out.println("-----------------------------------------------------------------------");
+    }
+
+
+    private static void listCondaInstallations() {
+        System.out.println("\n----------------------------CONDA INSTALLS-----------------------------");
+        System.out.print(
+                findCondaInstallations().stream()
+                        .map(PythonPathsCommand::formatCondaInstallation)
+                        .collect(Collectors.joining(System.lineSeparator()))
+        );
+        System.out.println("-----------------------------------------------------------------------");
+    }
+
+    private static String formatPythonInstallation(PythonDetails pythonDetails) {
+        return formatPythonInstallation(pythonDetails, 1);
+    }
+
+    private static String formatPythonInstallation(PythonDetails pythonDetails, int numberOfTabs) {
+        String tabs = IntStream.range(0, numberOfTabs).mapToObj(index -> "\t").collect(Collectors.joining(""));
+        return String.format(" -%s%s: %s%n%spath: %s%n%sversion: %s",
+                "\t",
+                numberOfTabs > 1 ? "name" : "id",
+                pythonDetails.id(),
+                tabs,
+                pythonDetails.path(),
+                tabs,
+                pythonDetails.version());
+    }
+
+    private static String formatCondaInstallation(CondaDetails condaDetails) {
+        List<String> formattedCondaEnvironments = new ArrayList<>();
+
+        condaDetails.environments().forEach(pythonDetails ->
+                formattedCondaEnvironments.add(formatPythonInstallation(pythonDetails, 2)));
+
+        return String.format(" -\tid: %s%n\tpath: %s%n\tversion: %s%s",
+                condaDetails.id(),
+                condaDetails.path(),
+                condaDetails.version(),
+                String.format(
+                                "\t--------------------------ENVIRONMENTS-------------------------%n" +
+                                "\t%s" +
+                                "\t---------------------------------------------------------------%n",
+                        String.join(System.lineSeparator() + "\t", formattedCondaEnvironments)
+                )
+        );
+    }
+
+    private static void listVenvInstallations() {
```

Comment:
Throw an exception here so we know it's not supported yet.

Reply:
Great idea

Reply:
Adding them won't be much work so I'll complete them in this PR as well